### ISSUE TITLE
update asyncio.async() to use asyncio.ensure_future()

### ISSUE THREAD

**Issue by FirefighterBlu3** [aio-libs/janus#6]
```python
  File "/usr/lib/python3.5/site-packages/janus-0.2.1-py3.5.egg/janus/__init__.py", line 147, in task_maker
    task = asyncio.async(f(), loop=self._loop)
  File "/usr/lib/python3.5/asyncio/tasks.py", line 530, in async
    DeprecationWarning)
  File "/var/bluelabs/python/wampcams.py", line 24, in warn_with_traceback
    traceback.print_stack()
/usr/lib/python3.5/asyncio/tasks.py:530: DeprecationWarning: asyncio.async() function is deprecated, use ensure_future()
  DeprecationWarning)
```

**Comment by FirefighterBlu3** [comment]
```diff
--- janus/__init__.py.orig	2016-07-06 05:17:40.000000000 +0000
+++ janus/__init__.py	2016-07-07 04:52:08.375292307 +0000
@@ -128,7 +128,7 @@
                 self._async_not_empty.notify()
 
         def task_maker():
-            task = asyncio.async(f(), loop=self._loop)
+            task = asyncio.ensure_future(f(), loop=self._loop)
             task.add_done_callback(self._pending.discard)
             self._pending.add(task)
 
@@ -144,7 +144,7 @@
                 self._async_not_full.notify()
 
         def task_maker():
-            task = asyncio.async(f(), loop=self._loop)
+            task = asyncio.ensure_future(f(), loop=self._loop)
             task.add_done_callback(self._pending.discard)
             self._pending.add(task)

```

**Comment by asvetlov** [comment]
fixed in janus==0.2.2
